cmd/lakefs/cmd/run.go

```diff
@@ -220,7 +220,9 @@ var runCmd = &cobra.Command{
 		httputil.SetHealthHandlerInfo(metadata.InstallationID)
 
 		// start API server
-		otfDiffService, closeOtfService := tablediff.NewService()
+		hd, _ := os.UserHomeDir()
+		defaultPluginsPath := fmt.Sprintf("%s/.lakefs/plugins/", hd)
```

Comment:
Can we use the defaults file under config for default config parameters? If you embed them here it's easy to use something else as the default in other locations.

Reply:
It's just that the default value for each plugin location will be the same, meaning I'll have to set the same value for each possible plugin.
I'll move the functionality to the config defaults file, but I'll explicitly call it to retrieve the value.